setup.sh

```diff
@@ -38,8 +38,12 @@ if [[ "$OS" == "Fedora" ]]; then
 elif [[ "$OS" == "Ubuntu" ]] || [[ "$OS" == "LinuxMint" ]]; then
   sudo apt-get install ffmpeg python-imdbpy python-notify2
   sudo apt-get install python-dbus libssl-dev libffi-dev
-  sudo -H pip install -r requirements.txt
   sudo apt-get install chromium-chromedriver
+  sudo pip install virtualenv
+  virtualenv env
+  virtualenv -p /usr/bin/python2.7 env
+  source env/bin/activate
+  pip install -r requirements.txt
```

Comment:
you forgot sudo here

Reply:
`sudo` in which line?

Reply:
46

Reply:
`sudo` will install packages systemwide but we need to install packages in virtual environment

Reply:
Yeah you are right. The problem is that some packages need sudo to install. You can try something like [that](https://askubuntu.com/questions/234758/how-to-use-a-python-virtualenv-with-sudo)

Reply:
As pointed out in the article, if we need an external script to run as superuser, we have to follow that method, but pip is already created once we create a virtual environment.